CMakeLists.txt

```diff
@@ -409,6 +409,12 @@ set(SOUPER_PASS_PROFILE_ALL ${CMAKE_BINARY_DIR}/${CMAKE_SHARED_LIBRARY_PREFIX}so
 set(PROFILE_LIBRARY ${CMAKE_BINARY_DIR}/${CMAKE_STATIC_LIBRARY_PREFIX}profileRuntime${CMAKE_STATIC_LIBRARY_SUFFIX})
 configure_file(${CMAKE_SOURCE_DIR}/utils/reduce.in ${CMAKE_BINARY_DIR}/reduce @ONLY)
 configure_file(${CMAKE_SOURCE_DIR}/utils/cache_dump.in ${CMAKE_BINARY_DIR}/cache_dump @ONLY)
+configure_file(${CMAKE_SOURCE_DIR}/utils/cache_knownbits.in ${CMAKE_BINARY_DIR}/cache_knownbits @ONLY)
+configure_file(${CMAKE_SOURCE_DIR}/utils/cache_neg.in ${CMAKE_BINARY_DIR}/cache_neg @ONLY)
+configure_file(${CMAKE_SOURCE_DIR}/utils/cache_nonNeg.in ${CMAKE_BINARY_DIR}/cache_nonNeg @ONLY)
+configure_file(${CMAKE_SOURCE_DIR}/utils/cache_power.in ${CMAKE_BINARY_DIR}/cache_power @ONLY)
+configure_file(${CMAKE_SOURCE_DIR}/utils/cache_nonZero.in ${CMAKE_BINARY_DIR}/cache_nonZero @ONLY)
+configure_file(${CMAKE_SOURCE_DIR}/utils/cache_signbits.in ${CMAKE_BINARY_DIR}/cache_signbits @ONLY)
```

Comment:
it seems like these are all very similar scripts, can't they be collapsed into one?